### DISCUSSION TITLE
Vercel account has been blocked

### DISCUSSION THREAD

**Discussion by podukrnet** [vercel/community#3636] [Account Issue] [answered]
### Question

Banned vercel account through girhub because I forgot to pay for the version. Entering the office does not allow to pay for the version. Help

**Comment by swarnava** [comment]
Hi,

please create a Billing ticket using https://vercel.com/help

**Comment by podukrnet** [comment]
It is not entirely clear how to do this, because I can log into my personal account through git as before. It says the account is blocked. Tell me in more detail how it can be done? thank you

**Comment by swarnava** [comment, marked as answer]
I just checked your account and it seems blocked for violation our [Terms of Service](https://vercel.com/legal/terms) and [Fair Usage Policy](https://vercel.com/docs/concepts/limits/fair-use-policy). Unfortunately it cannot be enabled this time.
